src/firefly/js/tables/reducer/TableDataReduer.js

```diff
@@ -18,15 +20,18 @@ export function dataReducer(state={data:{}}, action={}) {
                 return update(root, { [tbl_id] : {selectInfo: {$set: selectInfo}}});
             } else return root;
 
+        case (Cntlr.TABLE_NEW_LOADED)  :
+            if (get(root, [tbl_id, 'tableMeta', 'Loading-Status'], 'COMPLETED') !== 'COMPLETED') {
+                return update(root, { [tbl_id] : {tableMeta: {['Loading-Status']:  {$set: 'COMPLETED'}}}});
```

Comment:
Minor: file name misses 'c' for reduCer.js ;-)

--
Emmanuel Joliet
NASA/IPAC  Infrared Science Archive
MS 100-22, Caltech, Pasadena, CA 91125
Office: 170
Email:  ejoliet@ipac.caltech.edu
Phone:  (626) 395-1489

> On Apr 23, 2016, at 11:53, Trey Roby <notifications@github.com> wrote:
> 
> In src/firefly/js/tables/reducer/TableDataReduer.js:
> 
> > @@ -18,15 +20,18 @@ export function dataReducer(state={data:{}}, action={}) {
> >                  return update(root, { [tbl_id] : {selectInfo: {$set: selectInfo}}});
> >              } else return root;
> >  
> > +        case (Cntlr.TABLE_NEW_LOADED)  :
> > +            if (get(root, [tbl_id, 'tableMeta', 'Loading-Status'], 'COMPLETED') !== 'COMPLETED') {
> > +                return update(root, { [tbl_id] : {tableMeta: {['Loading-Status']:  {$set: 'COMPLETED'}}}});
> could this be?
> return update(root, { [tbl_id] : {tableMeta: {Loading-Status: {$set: 'COMPLETED'}}}});
> 
> As I said earlier, I am begging to really not like update because it is so hard to read. I though you wrote something else to stick an object into a deep object?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>